src/components/layout/signed-in-user-navigation.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import MobileNavigationMenu from './mobile-navigation-menu';
+import UserNavigation from './user-navigation';
+import UserMenu from './user-menu';
+
+function SignedInUserNavigation(props) {
+  return (
+    <div>
+      <UserNavigation
+        messagesLabel={props.messagesLabel}
+        messagesLink={props.messageLink}
+        notificationsLabel={props.notificationsLabel}
+        notificationsLink={props.notificationsLink}
+      />
+      <UserMenu user={props.user} userMenuNavList={props.userMenuNavList} />
+      <MobileNavigationMenu isAdmin={props.isAdmin} />
```

Comment:
should I be able to send into MobileNavigationMenu a custom mobileNavList array and adminNavLink node if provided to SignedInUserNaviagation?

Reply:
Yes you should. Thanks for catching that